lib/test_queue/runner/rspec.rb

```diff
@@ -49,4 +34,50 @@ def summarize_worker(worker)
       end
     end
   end
+
+  class TestFramework
+    class RSpec < TestFramework
+      def all_suite_paths
+        options = ::RSpec::Core::ConfigurationOptions.new(ARGV)
+        options.parse_options if options.respond_to?(:parse_options)
+        options.configure(::RSpec.configuration)
+
+        ::RSpec.configuration.files_to_run.uniq
+      end
+
+      def suites_from_path(path)
+        ::RSpec.world.reset
```

Comment:
Does this lose output from previous suites if they've already been run? I would have expected this to clear the output buffer.